.github/CODEOWNERS

```diff
@@ -1114,3 +1114,9 @@
 
 #PRLabel: %CodeGen %Client
 /eng/scripts/typespec/                                             @JoshLove-msft @m-nash @jorgerangel-msft @jsquire
+
+# PRLabel: %Liftr-Lite
```

Comment:
You cannot add arbitrary labels to the repository.   Please work with @ronniegeraghty to get a service label assigned and centrally published.